config.yml

```diff
@@ -77,7 +77,11 @@ client:
 validators:
   - name: validator1
     bonded: 900000000upokt
+    # Config overrides for `validator1` only (primarily used on LocalNet).
+    # To change the default configs (that can be created using `poktrolld init` - adjust `cmd/poktrolld/cmd/config.go`)
     app:
+      # Ignite does not carry over all defaults, so we are going to match `minimum-gas-prices` with `cmd/config.go`.
```

Comment:
```suggestion
      # BUG_DEV_NOTE: Ignite does not carry over all defaults, so we are going to match `minimum-gas-prices` with `cmd/config.go`.
```

Reply:
This is not a bug, l'll leave DEV_NOTE though.